tests/Doctrine/Tests/Common/Persistence/ObjectManagerDecoratorTest.php

```diff
@@ -41,6 +50,7 @@ public function getMethodParameters()
             if ($method->getNumberOfParameters() != $method->getNumberOfRequiredParameters()) {
                 $methods[] = [$method->getName(), array_fill(0, $method->getNumberOfParameters(), 'all') ?: []];
             }
+            $methods[] = in_array($method->getName(), $voidMethods, true);
```

Comment:
This code doesn't make any sense, it adds a boolean as a data provider.

PHPUnit generates a warning for this:
```
1) Warning
The data provider specified for Doctrine\Tests\Common\Persistence\ObjectManagerDecoratorTest::testAllMethodCallsAreDelegatedToTheWrappedInstance is invalid.
```

See Travis: https://travis-ci.org/doctrine/common/jobs/197349731#L228